.vib/redis-cluster/goss/goss.yaml

```diff
@@ -1,39 +1,50 @@
+{{- $auth := printf "REDISCLI_AUTH=%s" .Vars.password }}
+{{- $headless_port := .Vars.redis.containerPorts.redis }}
+{{- $lb_endpoint := printf "-h redis-cluster -p %d" .Vars.service.ports.redis }}
+{{- $nodes := .Vars.cluster.nodes }}
 command:
-  redis-version:
-    exec: redis-server --version | grep --quiet "$APP_VERSION"
+  {{- $key := printf "key_%s" (randAlpha 5) }}
+  {{- $value := printf "value_%s" (randAlpha 5) }}
+  redis-set-key-value-pairs:
+    exec: |
+      export {{ $auth }} && \
+      redis-cli {{ $lb_endpoint }} CLUSTER SLOTS | sed -n '3p' | \
```

Comment:
As per [documentation](https://redis.io/commands/cluster-slots/), this command is now marked as deprecated. If you are trying to obtain the IP of the master node, you could try using `CLUSTER NODES` instead and then filter by role type.